opdracht1.js

```diff
@@ -5,32 +5,48 @@ const tvTypes = inventory.map((item) => {
     return item.type;
 });
 
-console.log(tvTypes);
+//console.log(tvTypes); <-- console.log van opdracht 1
 
 
 //Opdracht 1b
 //Maak een nieuwe array en filter op originalStock - Sold > 0
 //Indien true voeg toe aan nieuwe array
-const soldOutTvs = inventory.filter((item) => {
-    return (item.originalStock - item.sold) > 0;
-});
+//voor de bonusopdracht zet het geheel in een functie
+
+function getSoldOutTvs(){
+    const soldOutTvs = inventory.filter((item) => {
+        return (item.originalStock - item.sold) <= 0;
+    });
+    //console.log(soldOutTvs); <-- console.log van opdracht 1
+
+    return soldOutTvs;
+}
+
+
 
-console.log(soldOutTvs);
 
 //Opdracht 1c: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die over AmbiLight beschikken. Log de uitkomst in de console.
 
-const ambilightTvs = inventory.filter((item) => {
-    return item.options.ambiLight === true;
-});
+//gebruik de filter methode om alle tv's met ambilight op te halen
+//maak een nieuwe array en stop ze daarin
+//voor de bonusopdracht:  stop het geheel in een functie
+//return het resultaat
+
+function getAmbilightTvs() {
+    const ambilightTvs = inventory.filter((item) => {
+        return item.options.ambiLight === true;
+    });
+    //console.log(ambilightTvs); <-- de console.log van opdracht 1
+    return ambilightTvs;
+}
 
-console.log(ambilightTvs);
 
 
 //Opdracht 1d: Schrijf een functie die alle tv's van laagste naar hoogste prijs sorteert. Log de uitkomst in de console.
 
 //declareer een functie
 //gebruik de sort methode om op basis van prijs te sorteren
-function orderByPrice(inventory){
+function orderByPrice(){
 
     return inventory.sort((a , b) => {
```

Comment:
Je zou dit nog iets kunnen versimpelen zoals in het [voorbeeld van MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#description)
<img width="341" alt="Schermafbeelding 2022-07-08 om 14 36 23" src="https://user-images.githubusercontent.com/23332341/177993377-825c92e8-c13c-4325-9c7b-3a573f20bf32.png">